tests/unit/test.config.js

```diff
@@ -1029,6 +1029,23 @@ describe('config', () => {
       });
     });
 
+    it('finds a config without dot in your working directory', () => {
+      return withTempDir(async (tmpDir) => {
+        const lastDir = process.cwd();
+        process.chdir(tmpDir.path());
+        try {
+          const undotedConfig = path.resolve(
```

Comment:
nit, I guess this is a typo (which is not really a super big deal but thought to mention it anyway) `undotedConfig` => `undottedConfig`

Reply:
thanks